cmd/launcher/svc_config_windows.go

```diff
@@ -194,7 +194,18 @@ func checkRestartActions(logger *slog.Logger, service *mgr.Service) {
 
 // setRecoveryActions sets the recovery actions for the launcher service.
 // previously defined via wix ServicConfig Element (Util Extension) https://wixtoolset.org/docs/v3/xsd/util/serviceconfig/
-func setRecoveryActions(ctx context.Context, logger *slog.Logger, service *mgr.Service) {
+func checkRecoveryActions(ctx context.Context, logger *slog.Logger, service *mgr.Service) {
```

Comment:
Since we have `checkDelayedAutostart`, `checkDependOnService`, and `checkRestartActions`, calling this function `checkRecoveryActions` is more consistent. The `checkX` naming convention in this file is my fault 😅 -- I don't mind it, but I am not opposed to renaming the other three functions to `setX` or `checkAndSetX`.

If we keep the new `checkRecoveryActions` name, could you update the docblock to match?

Reply:
Names are hard LOL 
I feel that `checkX` is easy to understand, and since the service methods specifically say `SetX` after that check validation it is easy to infer that since the check "failed" the setting is now set.

Reply:
It's reasonable to follow convention here. As Becca says, please do update the comment

Reply:
I got it updated in [this commit ](https://github.com/kolide/launcher/pull/1859/commits/7bbd8a86ceed6ac2506f78041d4672a28542d5af)